backend/business-services/egov-hrms/README.md

```
@@ -0,0 +1,112 @@
+        2. There's no cap on the number of entries in the service history.
+        3. Captured as legacy data.
+
+**d) Educational Details:** Captures educational details of the employee. Captures information like Degree, Year of Passing, University, Specialization etc as part of the educational details.
+   
+   - **Constraints:**
+        1. Details pertaining to educational details like Degree, Specialization must be configured.
+
+
+**e) Departmental Tests:** Captures details of the tests undertaken by the employee. Like name of the test and year of passing.
+   
+   - **Constraints:**
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)